fe/fe-core/src/main/java/com/starrocks/analysis/StringLiteral.java

```diff
@@ -279,4 +285,18 @@ public void parseMysqlParam(ByteBuffer data) {
         data.get(bytes);
         value = new String(bytes);
     }
+
+    public static class StringLiteralFactory {
+        private StringLiteralFactory() {
+
+        }
+
+        public static StringLiteral buildStringLiteral(String value) {
```

Comment:
Why did you create a new Factory class? I think you can make buildStringLiteral as a StringLiteral static function. like
```
StringLiteral.create()
```